packages/manager/apps/pci-gateway/src/api/data/gateway.ts

```diff
@@ -0,0 +1,30 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+import { PaginationState } from '@ovhcloud/manager-components';
+import { Gateway, GatewayResponse } from '@/interface';
+
+export type GatewayOptions = {
+  pagination: PaginationState;
+};
+
+export const getAllAggregatedGateway = async (
+  projectId: string,
+): Promise<GatewayResponse> => {
+  const { data } = await v6.get<GatewayResponse>(
+    `/cloud/project/${projectId}/aggregated/gateway`,
+  );
+  return data;
+};
+
+export const paginateResults = (
+  items: Gateway[],
```

Comment:
```suggestion
  items: unknown[],
```

It doesn't seem specific to Gateways
We could put this function next to the datagrid in the manager-components maybe